CoMD/src-threaded/ljForce.cpp

```diff
@@ -218,8 +229,13 @@ int ljForce(SimFlat* s)
              for (int m=0; m<3; m++)
              {
                dr[m] *= fr;
+#ifdef DO_CUDA
```

Comment:
I understand this is where you needed to use atomic to make the implementation correct since there is a race condition.  I’d also like to see the reduction in there (commented out for now) that we will use once we get rid of the race condition.